wikipedia-dark.user.css

```
@@ -5547,6 +5547,21 @@ regexp("^https://(foundation|donate)\.wikimedia\.org.*$") {
     background: var(--brown-4) !important;
   }
 }
+@-moz-document domain("de.wiktionary.org") {
+  div.mw-parser-output h2, div.mw-parser-output h3 {
+    background-color: transparent !important;
+  }
+  div[style*="background-color: #EFF5FB"],
+  div[style*="background-color:#EAF3FC"],
+  table[style*="background-color:#F5FAFE"],
+  td[style*="background-color: #F7F7F9"] {
+      background-color: transparent !important;
```

```suggestion
      background-color: var(--gray-2_75) !important;
```

That is more like it, transparent is not ideal.